src/app/services/count.service.spec.ts

```diff
@@ -43,4 +44,39 @@ describe('CountService', () => {
         done();
       });
   });
+
+  it('should request counts if non subject set aware constraint', () => {
+    let resourceServiceCounts = spyOn(countService['resourceService'], 'getCounts');
```

Comment:
You can also specify `resourceService = TestBed.get(ResourceService);` in the `beforeEach` block, just as for the count service, and `spyOn(resourceService, 'getCounts')`.